a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -1350,10 +1350,16 @@ <h4>Examples</h4>
 								data-localization-id="charts-diagrams-formulas-extended"
 								data-localization-mode="descriptive">Charts, diagrams, figures, and graphs have extended descriptions</span>
 						</li>
-						<li>
+<li><span data-localization-id="additional-accessibility-information-adaptation-closed-captions" data-localization-mode="descriptive">
+Videos included in publications have closed
+							captions </span></li>
+
+<li><span data-localization-id="additional-accessibility-information-adaptation-open-captions" data-localization-mode="descriptive">
+Videos included in publications have open captions </span></li>
+<li>
 							<span data-localization-id="charts-diagrams-formulas-unknown"
 								data-localization-mode="descriptive">
-								math, chemical formulas, charts, diagrams, figures, and graphs not identified as being accessible</span>
+								math, chemical formulas, charts, diagrams, figures, graphs, and videos not identified as being accessible</span>
```

Comment:
What happens if only some of these things are not identified as being accessible? I wouldn't think it'd happen too often that they're all found in a book and they're all not identified as accessible. This relates to the problem with the techniques that I noted in #505 that the statement about all rich content types not being identified could be output at the same time as statements about individual rich content being accessible.